be/btree_internal.h

```diff
@@ -34,13 +34,15 @@
 #include "format/format.h" /* m0_format_header */
 #include "be/btree.h"      /* BTREE_FAN_OUT */
 #include "be/btree_xc.h"   /* m0_be_btree_backlink_xc */
+#include "lib/types.h"
 
 /* btree constants */
 enum {
 	KV_NR = 2 * BTREE_FAN_OUT - 1,
 };
 
 struct be_btree_key_val  {
+	char inlkey[64];
```

Comment:
alignement.

Reply:
Ok.
I will update it.

Reply:
Can we use macro instead of magic number ?

Reply:
@jamohite Instead of 64? Yes, I will update that.